django_fido/management/commands/download_authenticator_metadata.py

```diff
@@ -0,0 +1,54 @@
+"""Command to download metadata for authenticators."""
+from __future__ import unicode_literals
+
+import json
+from base64 import b64decode
+
+import requests
+from django.core.management.base import BaseCommand, CommandError
+from jwcrypto.jwk import JWK
+from jwcrypto.jwt import JWT
+
+from django_fido.models import AuthenticatorMetadata
+from django_fido.settings import SETTINGS
+
+FIDO_MDS_URL = 'https://mds2.fidoalliance.org/'
```

Comment:
I think this value should be configurable in settings as well. The current URL can be the default value.